src/modals/ClaimReward/index.tsx

```diff
@@ -103,7 +103,7 @@ export const ClaimReward = () => {
           <FooterWrapper>
             <div>
               <ButtonSubmit
-                text={`Submit${submitting ? 'ting' : ''}`}
+                text={`${t('submit')}${submitting ? t('ting') : ''}`}
```

Comment:
Does `i18n` have a feature that we can use to combine these two like we have done with `_one` and `_other`. Or just have 2 separate keys for `submit` and `submitting`.